pyproject.toml

```diff
@@ -51,7 +51,7 @@ dependencies = [
 
 [project.optional-dependencies]
 docs = ["sphinx", "sphinx-rtd-theme"]
-testing = [
```

Comment:
While I was here, I made it in line with other AiiDA repos that use `tests` extras.